cmd/ipfs-cluster-service/main.go

```diff
@@ -213,6 +213,31 @@ configuration.
 				cfgMgr, cfgs := makeConfigs()
 				defer cfgMgr.Shutdown() // wait for saves
 
+				var alreadyInitialized bool
+				if _, err := os.Stat(configPath); !os.IsNotExist(err) {
+					alreadyInitialized = true
+				}
+
+				if alreadyInitialized {
+					// acquire lock for config folder
+					err := locker.lock()
+					checkErr("acquiring execution lock", err)
+					defer locker.tryUnlock()
+
+					if !c.Bool("force") {
```

Comment:
I would like these prompt messages to be defined in a constant so that we can do:

```go
if !c.Bool("force") && !yesNoPrompt(stateCleanupPrompt) {
  return nil
}
```